test/unit/util-specs.js

```diff
@@ -1,10 +1,20 @@
-import {DOMParser} from '@xmldom/xmldom';
 import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import sinon from 'sinon';
 import xpath from 'xpath';
 
-import {addVendorPrefixes, getOptimalXPath, xmlToJSON} from '../../app/renderer/util';
+import {
+  addVendorPrefixes,
+  areAttrAndValueUnique,
+  domParser,
+  findDOMNodeByPath,
+  findJSONElementByPath,
+  getOptimalClassChain,
```

Comment:
not sure these methods should all be in util
They all serve particular purpose (finding locators) and it makes sense to have them isolated in a separate module/class (maybe multiple)

Reply:
same applies to tests

Reply:
Yeah, I was considering some separation of the utils file as well. Say, one file for the source conversion (XML/JSON/Document), one for the locator generation, and one with everything else (currently only `withTranslation` and `addVendorPrefixes`)